src/main/java/fr/sncf/osrd/train/TrackSectionRange.java

```diff
@@ -99,6 +99,17 @@ public static TrackSectionRange makeNext(
         return new TrackSectionRange(edge, direction, edge.length, Double.max(edge.length - desiredLength, 0));
     }
 
+    /** Build a track section range given a direction and a desired length. */
+    public static TrackSectionRange makePrev(
+            TrackSection edge,
+            EdgeDirection direction,
+            double desiredLength
+    ) {
```

Comment:
Can we simplify this function using `makeNext` function?
```java
return makeNext(edge, direction.opposite(), desiredLength);
```

Reply:
Yes you're right.